js/src/Collection/TodoListCollection.js

```diff
@@ -0,0 +1,13 @@
+var Backbone = require('backbone');
+
+var TodoListModel = require('../Model/TodoListModel.js');
+
+var TodoListCollection = Backbone.Collection.extend({
+    model: TodoListModel,
+    url: '/todolists',
+    parse: function(response){
+        return response.result;
+    }
+})
```

Comment:
Needs semi-colon. Things like if statements, functions, for loops don't need a semi-colon, but `expressions`, such as `var x = 1;`, `return response.result;`, do need one.